lessons/Early-Math/commutative-property/langs/en-us/pages/discover-comm-cont.adoc

```diff
@@ -0,0 +1,68 @@
+= Discover the Commutative Property - Part 2
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Analyze and evaluate the Circles below to decide if the Commutative Property holds for problems involving *addition*.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(+ (+ 9 2) 8))}	| @show{(coe  '(+ (+ 8 2) 9))}
```

Comment:
I would describe the example here as illustrating the Associative property because you have grouped things differently.

Strongly suggest using (9+2)+8 and 8 + (9+2) instead.

Reply:
The bottom half of the worksheet is subtraction? (And always has been? Maybe it didn't build for some reason? Hoping you can see it.

(Agree with other feedback and responding to it.)

Reply:
Tried to implement this feedback. Take a look and see if it feels better to you!